save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/FiltersRow.kt

```diff
@@ -176,3 +193,50 @@ private fun testExecutionFiltersRow(
         }
     }
 }
+private fun nameFiltersRow(
+) = FC<NameFilterRowProps> { props ->
+
+    val (filtersName, setFiltersName) = useState(props.name)
```

Comment:
Maybe keep the whole filters object in state?

Reply:
The project and organization have different filter objects. And here I wanted to make a unique component for filtering by string.